radlibDemoApp/www/index.html

```diff
@@ -43,7 +43,7 @@ <h1 class="topcoat-navigation-bar__title">RadLib Demo</h1>
                         </a>
                     </li>
                     <li class="topcoat-list__item side-nav__list__item is-active">
-                        <a href="#" class="side-nav__button" onclick="alert('Delete Database clicked')">
+                        <a href="#" class="side-nav__button" onclick="db_clear()">
```

Comment:
looks good to me! If someone else can go through this too and merge it, it'd be lovely :+1: